lib/swoosh/application.ex

```diff
@@ -7,15 +7,15 @@ defmodule Swoosh.Application do
     import Supervisor.Spec, warn: false
 
     children = [
-      worker(Swoosh.InMemoryMailbox, []),
+      worker(Swoosh.Adapters.Local.Storage.Memory, []),
     ]
 
     children =
       if Application.get_env(:swoosh, :serve_mailbox) do
```

Comment:
Could be extracted to function, potentially two with one overload just returning `children`.  Not strongly against this though.